tuftool/src/root.rs

```diff
@@ -327,12 +333,23 @@ impl Command {
 
         // Quick check that root is signed by enough key IDs
         for (roletype, rolekeys) in &signed_root.signed().signed.roles {
-            if rolekeys.threshold.get() > rolekeys.keyids.len() as u64 {
-                return Err(error::Error::UnstableRoot {
-                    role: *roletype,
-                    threshold: rolekeys.threshold.get(),
-                    actual: rolekeys.keyids.len(),
-                });
+            let threshold = rolekeys.threshold.get();
+            let keyids = rolekeys.keyids.len();
+            if threshold > keyids as u64 {
+                // Return an error when the "force" flag wasn't set
+                if !force {
+                    return Err(error::Error::UnstableRoot {
+                        role: *roletype,
+                        threshold: threshold,
+                        actual: keyids,
+                    });
+                }
+                // Print out a warning letting the user that the generated file isn't complient with
```

Comment:
spelling: "compliant"

Can we clarify in the comments the difference between this check/warning and the one a few lines down?

Reply:
I see the top check was clarified to `Quick check that root is signed by enough key IDs, in all its roles`, but I'm still not sure what the lower check is for, that says `Signature check for root`?